### PR TITLE
#78965 Adding the code-block to prevent the translation

### LINKED ISSUE #78965: Docs: Add code-block wrappers to code examples in ignores.rst.
### Summary

**Problem**:
Throughout the Ansible docs, there are instances where example code is preceded with a lead-in sentence ending in `::`.
Translation programs then attempt to translate this code, which we don't want.

**Solution:**
Enclose code  in a `.. code-block:: <lexer>` element, so that translation processes know to skip this content.
For a list of allowed values for _`<lexer>`_ , refer to [Syntax highlighting - Pygments](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#syntax-highlighting-pygments).

**Scope:**
In the `ignores.rst ` file in the Developer Guide (`docs/docsite/rst/dev_guide/testing/sanity/`), there are 2 instances of lead-in sentences ending with `::`. Use the following `grep` command to identify the files and line numbers:

```
$ grep -rn --include "*.rst" "^[[:blank:]]*[^[:blank:]\.\.].*::$" . | grep ignores.rst
```

**Example:**

Before:
```
Before running ``ansible-playbook``, run the following command to enable logging::

    export ANSIBLE_LOG_PATH=~/ansible.log

```

After:
```
Before running ``ansible-playbook``, run the following command to enable logging:

.. code-block:: shell

    export ANSIBLE_LOG_PATH=~/ansible.log

```

This problem has been addressed in some other guides; view these merged PRs to help get you started:
- Network Guide: [#75850](https://github.com/ansible/ansible/pull/75850/files) 
- Developer Guide: [#75849](https://github.com/ansible/ansible/pull/75849/files) 

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/dev_guide/testing/sanity/ignores.rst

### Ansible Version

```console
n/a
```

### Configuration

```console
n/a
```

### OS / Environment

n/a

### Additional Information

When example code is enclosed within a code-block element, translation programs do not attempt to translate the code.

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by Abelarm**
##### SUMMARY
Added the code block to prevent the translation to the ignores.rst #78965 
Fixes #78965

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
/docs/docsite/rst/dev_guide/testing/sanity/ignores.rst

### CODE REVIEW

File: docs/docsite/rst/dev_guide/testing/sanity/ignores.rst [hunk 1]
```diff
@@ -71,7 +71,9 @@ This will be the name of the sanity test.
 If the sanity test is specific to a version of Python, the name will include a dash (``-``) and the relevant Python version.
 If the named test uses error codes then the error code to ignore must be appended to the name of the test, separated by a colon (``:``).
 
-Below are some example ignore entries for an Ansible collection::
+Below are some example ignore entries for an Ansible collection:
+
+.. code-block::
```
Comment (ariordan-redhat) [inline docs/docsite/rst/dev_guide/testing/sanity/ignores.rst]:
```suggestion
.. code-block:: text
```

File: docs/docsite/rst/dev_guide/testing/sanity/ignores.rst [hunk 2]
```diff
@@ -81,7 +83,9 @@ It is also possible to skip a sanity test for a specific file.
 This is done by adding ``!skip`` after the sanity test name in the second column.
 When this is done, no error code is included, even if the sanity test uses error codes.
 
-Below are some example skip entries for an Ansible collection::
+Below are some example skip entries for an Ansible collection:
+
+.. code-block::
```
Comment (ariordan-redhat) [inline docs/docsite/rst/dev_guide/testing/sanity/ignores.rst]:
```suggestion
.. code-block:: text
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/dev_guide/testing/sanity/ignores.rst (hunk 1 above)
Comment (Abelarm) [reply docs/docsite/rst/dev_guide/testing/sanity/ignores.rst:76]: I was not sure about the type of the lexer to use to I went with the empty one...
thank you for you suggestion

**Comment by ariordan-redhat** [comment]
Hi @Abelarm 
Thanks for submitting this PR.
Could you add the `text` lexer to the `code-block` line, please?  
The ignore file is a `.txt` file, so the `text` lexer works here.

**Comment by Abelarm** [comment]
> Hi @Abelarm Thanks for submitting this PR. Could you add the `text` lexer to the `code-block` line, please? The ignore file is a `.txt` file, so the `text` lexer works here.

I was not really sure about what kind of lexer to use so I went with the emtpy one

**Review by ariordan-redhat — APPROVED** [review]
Docs build looks good to me

**Comment by ariordan-redhat** [comment]
@Abelarm Yes, it's not always obvious which lexer to use.
I built the docs and your changes look good: PR approved.

Thank you for contributing to the Ansible docs!
We'll be posting more small docs issues through Hacktoberfest.

**Comment by Abelarm** [comment]
> @Abelarm Yes, it's not always obvious which lexer to use.
> 
> I built the docs and your changes look good: PR approved.
> 
> 
> 
> Thank you for contributing to the Ansible docs!
> 
> We'll be posting more small docs issues through Hacktoberfest.
> 
> 

I don't have a lot of experience on Ansible but I can help with doc and pure Python issue!! Gonna keep checking the issues

**Pull request merged**